### PR TITLE
[s] Fixes an issue with nuke cores and AIs 

### PR DESCRIPTION
**Pull request by Contrabang**
## What Does This PR Do

Makes sure that the user is adjacent to it, because for some reason AI calls attack_hand on nuclear bombs. Its not really that game breaking, just funny.

## Why It's Good For The Game
Fixes an oversight

## Testing

I could no longer use my remote-AI hands to touch the nuclear bomb

## Changelog
:cl:
fix: AIs can no longer remove the nuclear core from the nuclear bomb
/:cl:

### CODE REVIEW

**Review by Burzah — APPROVED** [review]

File: code/game/gamemodes/nuclear/nuclearbomb.dm [hunk 1]
```diff
@@ -342,6 +342,8 @@ GLOBAL_VAR(bomb_set)
 /obj/machinery/nuclearbomb/attack_hand(mob/user as mob)
```
Comment (DGamerL) [inline code/game/gamemodes/nuclear/nuclearbomb.dm:342]: Nitpick, but do we still need `as mob` here? I thought it was mostly deprecated except for some things like verbs?

**Review by DGamerL — APPROVED** [review]
LGTM

File: code/game/gamemodes/nuclear/nuclearbomb.dm (hunk 1 above)
Comment (Contrabang) [reply code/game/gamemodes/nuclear/nuclearbomb.dm:342]: Not really needed, but I want really want to expand the scope of this PR

Comment (DGamerL) [reply code/game/gamemodes/nuclear/nuclearbomb.dm:342]: Fair, feel free to keep it in then, fine to stay for a future PR

**Review by farie82 — APPROVED** [review]

**Pull request merged**